x-pack/plugins/cases/public/components/user_actions/index.test.tsx

```diff
@@ -285,81 +287,41 @@ describe(`UserActions`, () => {
     });
   });
 
-  it('calls update description when description markdown is saved', async () => {
```

Comment:
Moved this test to `x-pack/plugins/cases/public/components/user_actions/use_description_handler.test.tsx`